engine/src/star_system_xml.cpp

```diff
@@ -121,36 +121,37 @@ static bool EvalCondition( const char *op, const T &left, const T &right )
     }
 }
 
-static bool ConfigCondition( const string &cond )
-{
-    if ( cond.empty() )
-        return true;
-    char varname[64];
-    char op[3];
-    char varval[64];
-    bool ok = 3 == sscanf( cond.c_str(), "%63[-a-zA-Z_0-9] %2[<>=!] %63[-0-9.Ee]",
-                           varname,
-                           op,
-                           varval );
-    if (!ok)
-        return false;
-    //finalize character strings, for security
-    varname[sizeof (varname)/sizeof (*varname)-1] = 0;
-    op[sizeof (op)/sizeof (*op)-1] = 0;
-    varval[sizeof (varval)/sizeof (*varval)-1]    = 0;
-    //try to parse varval - if not parseable as float, assume it's a string
-    char *endptr = 0;
-    bool  rv;
-    if (endptr == varval) {
-        string sval = vs_config->getVariable( "graphics", varname, "0.0" );
-        rv = EvalCondition< string > ( op, sval, varval );
-    } else {
-        float fval = XMLSupport::parse_floatf( vs_config->getVariable( "graphics", varname, "0.0" ) );
-        float fref = XMLSupport::parse_floatf( varval );
-        rv = EvalCondition< float > ( op, fval, fref );
-    }
-    return rv;
-}
+// Commented out 2020-08, issue #239
```

Comment:
Can you elaborate on this? why are we commenting a whole function out?

Reply:
@nabaco because it's not used anywhere, except in code that is also commented out. See #239 for further details.

Reply:
Turns out that `ConfigCondition()` _was_ used prior to #181 . Hmm. We might want to look at this a little more closely.

Reply:
Hmm. The string "ConfigCondition" occurs exactly twice throughout the entire codebase. 
Once here, where it is defined, and once in system_factory.cpp, where it is commented out. Where was it used?

Reply:
@evertvorster I put more details in #239 .

Reply:
@stephengtuggy true, it was used in a few places pre #181. Not so much afterwards.
@royfalk what do you think about this function. Is it used anywhere, or can we finally rip it out?

Reply:
I think we can leave it commented out for now, but not delete it entirely. More details in #239 discussion.